### PR TITLE
issue#2527 fixed - Munu duplication error

### PR DESCRIPTION
**Pull request by suhhee1011**
## Type of Change

- [x] **Bugfix**: Change which fixes an issue
- [ ] **New Feature**: Change which adds functionality
- [ ] **Documentation Update**: Change which improves documentation
- [ ] **UI**: Change which improves UI

## Description

- Remove duplication of the sidebar menu items.
image

## Checklist

- [x] **Quality**: This PR builds and passes our npm test and works locally
- [x] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [x] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not (if applicable)
- [ ] **Documentation**: This PR includes updated/added documentation to user exposed functionality or configuration variables are added/changed or an explanation of why it does not(if applicable)

### CODE REVIEW

**Review by HyperTHD — DISMISSED** [review]

**Comment by HyperTHD** [comment]
Can believe I unintentionally didn't catch this too. Nice find

File: src/api/status/public/index.html [hunk 1]
```diff
@@ -355,66 +303,17 @@ <h4 class="mb-0" id="total-contributors-satellite">&nbsp;</h4>
...
-                <div class="text-end pt-1">
-                  <h6 class="mb-0 text-capitalize">Telescope</h6>
-                  <p class="text-sm mb-0 text-capitalize">Total feeds</p>
-                  <h4 class="mb-0" id="feeds-count">&nbsp;</h4>
-                </div>
-              </div>
-              <hr class="dark horizontal my-0" />
-              <div class="card-footer p-3"></div>
-            </div>
-          </div>
```
Comment (Andrewnt219) [inline src/api/status/public/index.html:415]: Why are these removed? They are cards for total posts and feeds.

File: src/api/status/public/index.html [hunk 2]
```diff
@@ -355,66 +303,17 @@ <h4 class="mb-0" id="total-contributors-satellite">&nbsp;</h4>
...
+                      <a href="" title="User" target="_blank" id="latest-author-satellite"
                         >&nbsp;</a
                       >
-                      <a
-                        href=""
-                        title="Commit"
-                        target="_blank"
-                        id="latest-author-commit-satellite"
-                        class="font-monospace btn-link text-xs"
-                      ></a>
```
Comment (Andrewnt219) [inline src/api/status/public/index.html:373]: This is to show the latest commit, don't delete this.

File: src/api/status/public/index.html [hunk 3]
```diff
@@ -270,22 +230,10 @@ <h4 class="mb-0" id="total-contributors-telescope">&nbsp;</h4>
...
                         >&nbsp;</a
                       >
-                      <a
-                        href=""
-                        title="Commit"
-                        target="_blank"
-                        id="latest-author-commit-telescope"
-                        class="font-monospace btn-link text-xs"
-                      ></a>
                     </p>
```
Comment (Andrewnt219) [inline src/api/status/public/index.html:237]: This is to show the latest commit, don't delete it.

**Review by Andrewnt219 — CHANGES_REQUESTED** [review]

File: src/api/status/public/index.html [hunk 4]
```diff
@@ -1178,6 +1077,6 @@ <h6 class="mt-3">Thank you for sharing!</h6>
     <!-- Control Center for Material Dashboard: parallax effects, scripts for the example pages etc -->
     <script src="assets/js/material-dashboard.min.js?v=3.0.0"></script>
     <!-- Telescope JS Files -->
-    <script type="module" src="js/pages/dashboard.js"></script>
+    <script src="js/github-stats.js"></script>
```
Comment (Andrewnt219) [inline src/api/status/public/index.html:1080]: `js/github-stats.js` itself doesn't do anything. It needs to attach to the load event inside `js/pages/dashboard.js`.

**Review by Andrewnt219 — CHANGES_REQUESTED** [review]

**Review by Kevan-Y — COMMENTED** [review]
Can this issue can be fixable in #2505? Since it is related to the sidebar.

File: src/api/status/public/index.html (hunk 3 above)
Comment (suhhee1011) [reply src/api/status/public/index.html:237]: Oh, I only removed the li tags in the sidebar. Maybe because my version was not the latest one then...I will add it

File: src/api/status/public/index.html (hunk 2 above)
Comment (suhhee1011) [reply src/api/status/public/index.html:373]: Oh, I only removed the li tags in the sidebar. Maybe because my version was not the latest one then...I will add it

File: src/api/status/public/index.html (hunk 1 above)
Comment (suhhee1011) [reply src/api/status/public/index.html:415]: Oh, I only removed the li tags in the sidebar. Maybe because my version was not the latest one then...I will add it

File: src/api/status/public/index.html (hunk 4 above)
Comment (suhhee1011) [reply src/api/status/public/index.html:1080]: Oh, I only removed the li tags in the sidebar. Maybe because my version was not the latest one then...I will add it
